src/styles/card.scss

```diff
@@ -0,0 +1,130 @@
+.container {
+  --column-count: 1;
+
+  display: grid;
+  gap: 46px 48px;
+  padding: 50px 40px;
+
+  grid-template-columns: repeat(var(--column-count), 200px);
+
+  justify-content: center;
+}
+
+@media (min-width: 488px) {
+  .container {
+    --column-count: 2;
+  }
+}
+
+@media (min-width: 768px) {
+  .container {
+    --column-count: 3;
+  }
+}
+
+@media (min-width: 1024px) {
+  .container {
+    --column-count: 4;
+  }
+}
+
+.card {
+  width: 200px;
+  border-radius: 5px;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  padding: 32px 16px;
+  padding-bottom: 16px;
+  transition-duration: 300ms;
+
+  &:hover {
```

Comment:
The `&:hover` effect causes the card to scale up, which might cause layout shifts. Ensure that scaling the cards does not affect the layout of surrounding elements or consider a different hover effect that does not cause reflow.